src/commands/index.ts

```diff
@@ -0,0 +1,95 @@
+import * as vscode from 'vscode';
+import { getConfig } from '../utils/vscode-extensions';
```

Comment:
It's common practice to always start the path with the current director a.k.a `./../utils/vscode-extensions`.

This means that this has to be changes everywhere were we have paths.